webdriver-spec.html

```diff
@@ -4319,7 +4319,7 @@ <h2 data-dfn-for="NavigatorAutomationInformation">Interface</h2>
 
 <p>A <dfn data-lt="keyboard-interactable">keyboard-interactable element</dfn>
  is any <a>element</a> that has a <a>focusable area</a>,
- or is the <a><code>body</code> element</a>,
+ is a <a><code>body</code> element</a>,
```

Comment:
The "body" that we link to is in the HTML5 spec. By definition it'll be an HTML document.